### PR TITLE
New pages

### PR DESCRIPTION
**Pull request by mikkelcodes**
To do:

- Responsiveness (several pages)
- Source videos from youtube (received credentials)
- Add Slider functionality
- Develop landing page and the rest of About page
- Replace inline styling with tailwind utility classes

### CODE REVIEW

File: packages/website/src/components/featuredTeam/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { EmployeeCard } from '../employeeCard';
+import Link from 'gatsby-link';
+import { Title } from '../title';
+import * as Button from '../button';
+
+export const FeaturedTeam = ({ team }) => {
+  const handleClick = (e) => {
+    return e;
```
Comment (TheKnarf) [inline packages/website/src/components/featuredTeam/index.js:9]: I don't think there's any need to return anything here?

File: packages/website/src/components/videoCard/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Link from 'gatsby-link';
+import Image from 'gatsby-image';
+
+export const VideoCard = ({ slug, title, thumbnail }) => {
+  const episode = title.split(' ')[0];
+  const onlyTitle = title.split(' ').slice(1).join(' ');
```
Comment (TheKnarf) [inline packages/website/src/components/videoCard/index.js:7]: Maybe better to do this on the calling side and let the component take in `episode` and `onlyTitle` as props instead?

File: packages/website/src/components/videoCard/index.stories.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+};
+
+export const Default = ({ ...args }) => <VideoCard {...args} />;
+Default.args = {
+  children: 'Child',
+  mainImage: { asset: { fluid: '' } },
+  slug: 'slug',
+  title: 'title',
+  tags: ['Ledelse'],
+  description: 'desc',
```
Comment (TheKnarf) [inline packages/website/src/components/videoCard/index.stories.js:15]: These args don't seem to match the props that VideoCard take?

File: packages/website/src/components/videoEpisode/index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                {subtitle}
+              </h4>
+
+              <h5 className="text-lg font-thin text-center tracking-wider mb-8">
+                {description}
+              </h5>
+            </div>
+            {/* ---- Above: Visible on Mobile only ---- */}
+            <div className="mb-8 sm:mb-8">
+              {/* <FeaturedVideo thumbnail={thumbnail} /> */}
```
Comment (TheKnarf) [inline packages/website/src/components/videoEpisode/index.js:24]: Remove outcommented code

File: packages/website/src/components/videoFilter/index.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  const [active, setActive] = useState(allVideos);
+  const [season, setSeason] = useState('1');
+
+  const updateVideos = (updatedVideos) => {
+    setActive(updatedVideos);
+  };
+
+  useEffect(() => {
+    const sortedActive = [];
+    active.forEach((el) => sortedActive.push(el));
```
Comment (TheKnarf) [inline packages/website/src/components/videoFilter/index.js:15]:
```
const sortedActive = [];
active.forEach((el) => sortedActive.push(el));
```

can be refactored to:

```
const sortedActive = [ ...active ];
````

File: packages/website/src/components/videoFilter/index.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    if (sort === 'oldest') {
+      sortedActive.sort((a, b) => {
+        if (a.publishedAt < b.publishedAt) {
+          return -1;
+        } else {
+          return 1;
+        }
```
Comment (TheKnarf) [inline packages/website/src/components/videoFilter/index.js:22]: Can be written using shorthand-if:

```
return (a.publishedAt < b.publishedAt) ? -1 : 1;
```

File: packages/website/src/components/videoFilter/index.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      });
+    } else if (sort === 'newest') {
+      sortedActive.sort((a, b) => {
+        if (a.publishedAt > b.publishedAt) {
+          return -1;
+        } else {
+          return 1;
+        }
```
Comment (TheKnarf) [inline packages/website/src/components/videoFilter/index.js:30]: Can be written using shorthand-if

File: packages/website/src/components/videoFilter/index.js [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+import { SortField } from '../blogFilter';
+
+export const VideoFilter = ({ allVideos, tabs }) => {
+  const [sort, setSort] = useState(null);
+  const [active, setActive] = useState(allVideos);
+  const [season, setSeason] = useState('1');
+
+  const updateVideos = (updatedVideos) => {
+    setActive(updatedVideos);
+  };
```
Comment (TheKnarf) [inline packages/website/src/components/videoFilter/index.js:11]: This function can probably be refactored away as it doesn't seem to do anything?

File: packages/website/src/components/videoFilter/index.js [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+  const seasonClick = (e) => {
+    setSeason(e.target.id);
+  };
+
+  return (
+    <div className="flex sm:flex-row flex-col justify-between sm:items-center sm:mb-10 mb-10">
+      {tabs ? (
+        <div className="flex tracking-wider items-center h-8 pt-6px sm:my-0 mt-4 mb-5">
+          {season && (
```
Comment (TheKnarf) [inline packages/website/src/components/videoFilter/index.js:50]: Consider moving season into its own component (in the same file).

File: packages/website/src/components/videoHero/index.js [hunk 10]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Image from 'gatsby-image';
+import { Title } from '../title';
+import { FeaturedVideo } from '../featuredVideo';
+
+export const VideoHero = ({ backgroundImage, thumbnail }) => {
+  return (
+    <div className="bg-navy w-full pt-2">
+      <div className="twelve:grid grid-cols-videos-hero mx-auto twelve:mb-24">
+        <div className="" />
```
Comment (TheKnarf) [inline packages/website/src/components/videoHero/index.js:10]: Why do we have a div with an empty className? Remove it.

File: packages/website/src/components/videoHero/index.js [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+        <span
+          className="relative twelve:-ml-20 twelve:pr-40 2xl:ml-0 2xl:pr-0 transform translate-x-2/7 twelve:translate-x-0 block"
+          style={{ maxHeight: '535px' }}
```
Comment (TheKnarf) [inline packages/website/src/components/videoHero/index.js:13]: Can the style be moved into a css-module?

File: packages/website/src/components/videoHero/index.js [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+        >
+          <Image
+            fluid={backgroundImage}
+            className="opacity-20"
+            style={{ maxHeight: '535px' }}
```
Comment (TheKnarf) [inline packages/website/src/components/videoHero/index.js:18]: Can the style be moved into a css-module?

File: packages/website/src/components/videoHero/index.js [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+          />
+          <div
+            className="absolute text-white top-1/2 transform -translate-y-1/2 -translate-x-40 md:m-0 ml-15"
+            style={{ top: '50%' }}
```
Comment (TheKnarf) [inline packages/website/src/components/videoHero/index.js:22]: Can the style be moved into a css-module?

File: packages/website/src/components/videoSection/index.js [hunk 14]
```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import { VideoFilter } from '../videoFilter';
+import { VideoCard } from '../videoCard';
+import * as Button from '../button';
+
+export const VideoSection = ({ allVideos, data, tabs }) => {
```
Comment (TheKnarf) [inline packages/website/src/components/videoSection/index.js:6]: Rename `allVideos` to `videoes`.

File: packages/website/src/components/videoSection/index.js [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+  // Placeholder until actual videos are loaded in
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [videos, setVideos] = useState(allVideos);
```
Comment (TheKnarf) [inline packages/website/src/components/videoSection/index.js:9]: What is this variable used for?

File: packages/website/src/components/videoSection/index.js [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+  const [visibleRows, setVisibleRows] = useState(3);
+
+  const handleViewMoreClick = () => {
+    setVisibleRows(visibleRows + 3);
+  };
+
+  const updateVideos = (videos) => {
+    setVideos(videos);
+  };
```
Comment (TheKnarf) [inline packages/website/src/components/videoSection/index.js:18]: This function seems useless as it doesn't do anything other than call setVideos directly. Refactor it away.

File: packages/website/src/components/videoSection/index.js [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+
+  useEffect(() => {
+    setVideos([1, 2, 3]);
```
Comment (TheKnarf) [inline packages/website/src/components/videoSection/index.js:21]: I don't understand the point of this code.

File: packages/website/src/pages/c-utvikler.js [hunk 18]
```diff
@@ -0,0 +1,44 @@
...
+import { NavyIntroImage } from '../components/navyIntroImage';
+import { BlogSlider } from '../components/blogSlider';
+import { FeaturedTeam } from '../components/featuredTeam';
+import { Testimonials } from '../components/testimonials';
+import { CtaSection } from '../components/ctaSection';
+
+const Systemutvikling = () => {
+  const data = useSubServiceQuery();
+  const employees = data.allSanityEmployee.edges.map((el) => el.node);
+  const team = employees.slice(0, 4);
```
Comment (TheKnarf) [inline packages/website/src/pages/c-utvikler.js:15]: `const employees` and `const team` could perhaps be moved into the GraphQl hook?

File: packages/website/src/pages/jobbe-i-alv.js [hunk 19]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Layout from '../components/layout';
+import { ImageTextShifted } from '../components/imageTextShifted';
+import { ImageTextListHero } from '../components/imageTextListHero';
+import { useWorkQuery } from '../hooks/useWorkQuery';
+import Image from 'gatsby-image';
+
+const WorkForAlv = () => {
+  const data = useWorkQuery();
+  console.log(data);
```
Comment (TheKnarf) [inline packages/website/src/pages/jobbe-i-alv.js:10]: Remove console.log.

File: packages/website/src/pages/kontakt-oss.js [hunk 20]
```diff
@@ -3,25 +3,31 @@ import Layout from '../components/layout';
...
   const [active, setActive] = useState('offer');
   const formChange = (e) => {
     setActive(e.target.id);
   };
+  const queryData = useContactQuery();
   return (
     <Layout path={location.pathname}>
       <div className="bg-navy">
-        <div className="bg-navy w-full sm:pb-12 pb-4 flex lg:flex-row flex-col -mt-16 lg:-mt-8 text-white max-w-1440 mx-auto">
+        {/* <div className="bg-navy w-full sm:pb-12 pb-4 flex lg:flex-row flex-col -mt-16 lg:-mt-8 text-white max-w-1440 mx-auto"> */}
```
Comment (TheKnarf) [inline packages/website/src/pages/kontakt-oss.js:21]: Remove outcommented code

File: packages/website/src/pages/om-oss.js [hunk 21]
```diff
@@ -0,0 +1,101 @@
...
+import { useAboutUsQuery } from '../hooks/useAboutUsQuery';
+import Image from 'gatsby-image';
+import { Title } from '../components/title';
+import { BlogSlider } from '../components/blogSlider';
+import { FeaturedTeam } from '../components/featuredTeam';
+
+const About = () => {
+  const data = useAboutUsQuery();
+  const employees = data.allSanityEmployee.edges.map((el) => el.node);
+  console.log(data);
```
Comment (TheKnarf) [inline packages/website/src/pages/om-oss.js:12]: Remove console.log

File: packages/website/src/pages/om-oss.js [hunk 22]
```diff
@@ -0,0 +1,101 @@
...
+  const team = employees.slice(0, 4);
```
Comment (TheKnarf) [inline packages/website/src/pages/om-oss.js:13]: `const employees` and `const team` can perhaps be moved into the GraphQL hook?

File: packages/website/src/pages/systemutvikling.js [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+import { WhatWeDo } from '../components/whatWeDo';
+import { RolesList } from '../components/rolesList';
+import { BlogSlider } from '../components/blogSlider';
+import { FeaturedTeam } from '../components/featuredTeam';
+import { Testimonials } from '../components/testimonials';
+
+const Systemutvikling = () => {
+  const data = useSubServiceQuery();
+  const employees = data.allSanityEmployee.edges.map((el) => el.node);
+  const team = employees.slice(0, 4);
```
Comment (TheKnarf) [inline packages/website/src/pages/systemutvikling.js:16]: `const employees` and `const team` can perhaps be moved into the GraphQL hook?

File: packages/website/src/assets/what_we_do_icon_box.svg [hunk 24]
```diff
@@ -0,0 +1,21 @@
+<svg id="Group_3905" data-name="Group 3905" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="91.973" height="80.704" viewBox="0 0 91.973 80.704">
```
Comment (TheKnarf) [inline packages/website/src/assets/what_we_do_icon_box.svg:1]: Maybe run all the svg files through https://jakearchibald.github.io/svgomg/?

File: packages/website/src/components/featuredTeam/index.js (hunk 1 above)
Comment (mikkelcodes) [reply packages/website/src/components/featuredTeam/index.js:9]: True, it's a placeholder. I should have commented it

File: packages/website/src/components/videoFilter/index.js (hunk 9 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoFilter/index.js:50]: Yep

File: packages/website/src/components/videoHero/index.js (hunk 11 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoHero/index.js:13]: yes

File: packages/website/src/components/videoCard/index.js (hunk 2 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoCard/index.js:7]: agree

File: packages/website/src/components/videoSection/index.js (hunk 15 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoSection/index.js:9]: This variable will receive playlist videos from youtube and update based on sorting selection.

File: packages/website/src/components/videoSection/index.js (hunk 16 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoSection/index.js:18]: updateVideos is passed to VideoFilter component as onChange (sorting). Does this make sense?

Comment (mikkelcodes) [reply packages/website/src/components/videoSection/index.js:18]: setVideos has been renamed to setSortedVideos

Comment (TheKnarf) [reply packages/website/src/components/videoSection/index.js:18]: Just pass setVideoen / setSortedVideoes directly

File: packages/website/src/components/videoSection/index.js (hunk 17 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoSection/index.js:21]: It is rather pointless until we have videos from YT. Again, I should have added a small comment abt that

File: packages/website/src/components/featuredVideo/index.stories.js [hunk 25]
```diff
@@ -0,0 +1,15 @@
...
+  title: 'components/featuredVideo',
+};
+
+export const Default = ({ ...args }) => <FeaturedVideo {...args} />;
+Default.args = {
+  children: 'Child',
+  mainImage: { asset: { fluid: '' } },
+  slug: 'slug',
+  title: 'title',
+  description: 'desc',
```
Comment (TheKnarf) [inline packages/website/src/components/featuredVideo/index.stories.js:14]: The args here doesn't seem to match the props that FeaturedVideo take..

File: packages/website/src/components/videoSection/index.js [hunk 26]
```diff
@@ -0,0 +1,66 @@
...
+    setVisibleRows(visibleRows + 3);
+  };
+
+  const updateVideos = (videos) => {
+    setSortedVideos(videos);
+  };
+
+  useEffect(() => {
+    setSortedVideos([1, 2, 3]);
+  }, []);
```
Comment (TheKnarf) [inline packages/website/src/components/videoSection/index.js:22]: This useEffect is deffinitily not needed. Remove it.

File: packages/website/src/components/videoFilter/index.js (hunk 8 above)
Comment (mikkelcodes) [reply packages/website/src/components/videoFilter/index.js:11]: This update has been made but after the current committed version. Should I push this or do we make it part of next round?

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**